.eslintrc.yml

```diff
@@ -98,6 +98,7 @@ rules:
   no-caller: error
   no-catch-shadow: error
   no-confusing-arrow: error
+  no-console: 'off'
```

Comment:
I don't think you need to disable this role. eslint seems to pass without this line since it lints `lib/` and `index.js` only and only the examples contain console.logs.